tests/snap_test.go

```diff
@@ -0,0 +1,244 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/canonical/matter-snap-testing/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	// enviroment variables
+	specificGpioChip = "GPIO_CHIP"
+	specificGpioLine = "GPIO_LINE"
+	gpioChipMock     = "MOCK_GPIO"
+)
+
+const snapMatterPiGPIO = "matter-pi-gpio-commander"
+const chipToolSnap = "chip-tool"
```

Comment:
Inconsistent naming.

Reply:
Changed `snapMatterPiGPIO` to `matterPiGPIOSnap`.